adoc/Harbor-Private-Registry-Installation.adoc

```diff
@@ -0,0 +1,804 @@
+
+.. Connect to an Azure Cache for Redis instance
++
+Add the following section to the `harbor-values.yaml` file and fill it with information reflecting the Azure Cache for Redis instance previously prepared.
+As mentioned above in the <<requirements-redis-azure>>, the address will have the form of `<azure-redis-cache>.redis.cache.windows.net`.
++
+.harbor-values.yaml
+[source,yaml]
+----
+redis:
+  type: external
+  external:
+    addr for redis: <host_redis>:<port_redis>
+    addr: "192.168.0.2:6379"
+----
+// TODO AI (Dirk Mueller) needs to be reviewed
+.. Connect to an Amazon ElastiCache Redis service
++
+Add the following section to the `harbor-values.yaml` file and fill it with information reflecting the Amazon ElastiCache Redis instance previously prepared:
++
+.harbor-values.yaml
+[source,yaml]
+----
+redis:
+  type: external
+  external:
+    addr for redis: <host_redis>:<port_redis>
+    addr: "192.168.0.2:6379"
+    password: securepassword
+----
+
+. Set up the passwords for deployment
+[#installation-passwords]
++
+By default, all passwords are automatically generated when installing {spr} with the Helm chart. They can be retrieved post-installation from the created {kube} secrets objects. For example, to retrieve the Harbor adminstrator password necessary to log in into the Harbor Portal UI as admin user, run this command after the deployment is finished:
++
+[source,bash]
+----
+kubectl get secret suse-registry-harbor-core -n registry -o jsonpath="{.data.HARBOR_ADMIN_PASSWORD}" | base64 --decode
+----
++
+To set a custom administrator password before the installation, modify your `harbor-values.yaml` file like this:
++
+.harbor-config-values.yaml
+[source,yaml]
+----
+harborAdminPassword: <password-for-admin-user>
+----
++
+Similarly, custom passwords may be set before the installation for the database and Redis services, if configured as internal services:
++
+.harbor-config-values.yaml
+[source,yaml]
+----
+database:
+  ...
+  internal:
+    password: <password-for-redis>
+
+redis:
+  ...
+  internal:
+    password: <password-for-redis>
+----
+
+. Finally, install {spr} with helm
++
+To install {spr} as a suse-registry release into the registry namespace with the custom configuration prepared in the `harbor-values.yaml` file in the previous steps, run the following command:
++
+[source,bash]
+----
+helm -n registry install suse-registry ./harbor -f harbor-values.yaml
+----
++
+Once the installation is complete, Helm will provide the information about the location of the newly installed registry, e.g.:
++
+[source,bash]
+----
+NAME: suse-registry
+LAST DEPLOYED: Fri Jul 24 10:34:53 2020
+NAMESPACE: registry
+STATUS: deployed
+REVISION: 1
+NOTES:
+Please wait for several minutes for Harbor deployment to complete.
+Then you should be able to visit the Harbor portal at https://core.harbor.domain // <1>
+----
+<1> You will see your `<ingress_url>` instead of `https://core.harbor.domain`.
```

Comment:
I think _<ingress_url>_ was actually replaced by _<core_fqdn>_ at the start of installation section...